docs/coverage/repositories/pull-requests.md

```diff
@@ -0,0 +1,55 @@
+---
+page_name: pull request
+file_name: pull-requests
+product: coverage
+---
+
+# [Coverage] Pull Requests page
+<!--TODO
+    Review page, see https://codacy.atlassian.net/browse/COV-119 for more details-->
+
+The **Pull Requests page** displays an overview of the pull requests in your repository, such as the analysis status and coverage metrics for each pull request. This allows you to monitor the coverage of the work in progress in your repository.
```

Comment:
Just wondering: is it still an "analysis" for coverage?

Reply:
I'm also a bit unsure. Here's a preview of the pull request list:

![image](https://user-images.githubusercontent.com/60105800/231519674-0f873ada-bf5f-483a-90ba-d7ddb9e6cbb9.png)

I'm changing this to reference "Status" only.